lib/vtls/vtls.c

```diff
@@ -878,11 +878,11 @@ bool Curl_ssl_cert_status_request(void)
 
 /*
  * Check whether the SSL backend supports false start.
+ * This API always returns false, as no SSL backends with support
+ * remain in-use.
  */
 bool Curl_ssl_false_start(void)
 {
-  if(Curl_ssl->false_start)
-    return Curl_ssl->false_start();
   return FALSE;
 }
```

Comment:
I believe this whole function can be removed now because nothing calls it, right?

Reply:
Good catch, you're right. Removed!